### PR TITLE
fix: external link proper spacing

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

https://app.clickup.com/t/pb9v24

Updates the external link component for using the proper space

### Before:
image

### After:
image

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [x] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

**Review by Highjhacker — APPROVED** [review]

File: resources/views/external-link-confirm.blade.php [hunk 1]
```diff
@@ -13,13 +13,13 @@
             Livewire.emit('openModal', '{{ $identifier }}')
         }
     }"
-    class="inline-block items-center space-x-2 font-semibold break-all cursor-pointer link"
+    class="inline-flex items-center font-semibold break-all cursor-pointer link"
```
Comment (ItsANameToo) [inline resources/views/external-link-confirm.blade.php:16]: doing that causes this (the icon no longer aligns with the text when it wraps)
image

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Comment by alfonsobries** [comment]
@ItsANameToo fixed 

image

File: resources/views/external-link-confirm.blade.php [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
     }"
-    class="inline-block items-center space-x-2 font-semibold break-all cursor-pointer link"
+    class="items-center font-semibold break-all cursor-pointer link"
     @click="openModal"
 >
     <span>{{ $text ?? $slot ?? '' }}</span>
 
     @unless($noIcon)
-        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 mr-2 ml-1 -mt-1" />
+        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 ml-1 -mt-2" />
```
Comment (ItsANameToo) [inline resources/views/external-link-confirm.blade.php:22]:
```suggestion
        <x-ark-icon name="link" size="sm" class="inline flex-shrink-0 ml-1 -mt-1" />
```

looks better with `-mt-1` I think 🤔 
image


with `-mt-2`:
image

Comment (alfonsobries) [reply resources/views/external-link-confirm.blade.php:22]: Changed for 1.5 
1: 
image

1.5:
image

Comment (alfonsobries) [reply resources/views/external-link-confirm.blade.php]: (mt-1 a little bit unaligned on multiline links)

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**